### PR TITLE
[nova]Add nova user to the EDPM host

### PR DESCRIPTION
**Pull request by gibizer**
This user will be used by nova-compute to copy data between compute nodes during migration.

### CODE REVIEW

**Comment by gibizer** [comment]
/unhold 
this works for me

**Review by slagle — APPROVED** [review]

File: roles/edpm_nova/molecule/default/verify.yml [hunk 1]
```diff
@@ -40,3 +40,17 @@
         - "Copying /var/lib/kolla/config_files/nova-blank.conf to /etc/nova/nova.conf"
         - "Copying /var/lib/kolla/config_files/01-nova.conf to /etc/nova/nova.conf.d/01-nova.conf"
         - "Copying /var/lib/kolla/config_files/02-nova-log.conf to /etc/nova/nova.conf.d/02-nova-log.conf"
+
+    - name: Lookup the nova user
+      ansible.builtin.shell:
```
Comment (gibizer) [inline roles/edpm_nova/molecule/default/verify.yml:45]: There is a comment from @ielod this in https://github.com/openstack-k8s-operators/edpm-ansible/pull/446#discussion_r1358369686 that I will fix quickly

**Comment by gibizer** [comment]
recheck
quay.io outage is over

**Comment by gibizer** [comment]
recheck
```
[WARNING]: ERROR: Can't pull image quay.io/podified-antelope-centos9/openstack-
[WARNING]: ERROR: Can't pull image quay.io/podified-antelope-centos9/openstack-
nova-libvirt:current-podified stderr: Trying to pull quay.io/podified-antelope-
nova-libvirt:current-podified stderr: Trying to pull quay.io/podified-antelope-
centos9/openstack-nova-libvirt:current-podified... Error: initializing source
centos9/openstack-nova-libvirt:current-podified... Error: initializing source
docker://quay.io/podified-antelope-centos9/openstack-nova-libvirt:current-
docker://quay.io/podified-antelope-centos9/openstack-nova-libvirt:current-
podified: reading manifest current-podified in quay.io/podified-antelope-
podified: reading manifest current-podified in quay.io/podified-antelope-
centos9/openstack-nova-libvirt: received unexpected HTTP status: 502 Bad
centos9/openstack-nova-libvirt: received unexpected HTTP status: 502 Bad
Gateway
Gateway
```

**Review by rebtoor — APPROVED** [review]

**Pull request merged**